lib/r2d2/google_payment_token.rb

```diff
@@ -0,0 +1,67 @@
+module R2D2
+  class GooglePaymentToken < PaymentToken
+    SignatureInvalidError = Class.new(PaymentToken::Error)
+    MessageExpiredError = Class.new(PaymentToken::Error)
+
+    HKDF_INFO = 'Google'
+
+    attr_reader :protocol_version, :recipient_id, :verification_keys, :signature, :signed_message
+
+    def self.to_length_value(*chunks)
+      chunks.flat_map do |chunk|
+        chunk_size = 4.times.map do |index|
+          (chunk.bytesize >> (8 * index)) & 0xFF
+        end
+        chunk_size.pack('C*') + chunk
+      end.join
+    end
+
+    def initialize(token_attrs, recipient_id:, verification_keys:)
```

Comment:
I believe this is Ruby v2.? syntax? If this is the only place we're using this it might be good to revert to "legacy" syntax to support a greater range of Ruby versions?

Reply:
Mandatory keyword args were introduced in Ruby 2.2, which was EOL'ed end of March after being released over 3 years ago. Given that we're dealing with credit card data, should we make the interface less clear to accommodate for old Ruby versions? Personally, I don't think we should.